students/jim_krecek/lesson03/mailroom.py

```diff
@@ -0,0 +1,46 @@
+data = [["Nate Secinaro",12,44,9],["Jess Reid",3],["Pat Carrier",65,41],["Zac Fisher",13,4,4],["Jim Krecek",3,33,57]]
+
+prompt = "\nWhat do you want to do?\n -1 Send a Thank You\n -2 Create a Report\n -3 Quit\n>>> "
+
+def send_ty():
+    askname = input("\nEnter the full name of the donor or type 'list' to see the list of donors:\n>>> ")
+    if askname.lower() == "list":
+        print("\nHere is a list of all of the donors:")
+        for d in data:
+            print(d[0])
+    
+    for d in data:
```

Comment:
this should be in the `else` block - you're looping every time when the user just enters `list` command.